index.html

```diff
@@ -11774,8 +11774,10 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
-				<p class="note">Authors are advised to use extreme caution and consider a wide range of disabilities when hiding visibly rendered content from assistive technologies. For example, a sighted, dexterity-impaired individual might use voice-controlled assistive technologies to access a visual interface. If an author hides visible link text "Go to checkout" and exposes similar, yet non-identical link text "Check out now" to the accessibility API, the user might be unable to access the interface they perceive using voice control software. Similar problems can also arise for screen reader users. For example, a sighted telephone support technician might attempt to have the blind screen reader user click the "Go to checkout" link, which they might be unable to find using a type-ahead item search ("Go to…"), since that text would have been hidden by the use of the attribute.</p>
+				<p>Authors MUST NOT use <code>aria-hidden</code> to hide the root element or the host language element that <a data-cite="html/dom.html#represents">represents</a> the contents of the primary document in view. For instance, the <code>html</code> or <code>body</code> elements in an HTML document. Authors MAY, with caution, use <sref>aria-hidden</sref> to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using <sref>aria-hidden</sref> to hide visible content MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies. Whenever possible it's strongly recommended authors use host language features to <a href="#tree_exclusion">exclude elements from the accessibility tree</a>.</p>
+				<p>Additionally, authors SHOULD ensure that any elements that are accessibility descendants of an aria-hidden element, or any elements which have been marked as aria-hidden themselves, are either prevented from receiving focus, or upon receiving focus use scripting to immediately move focus to another element.</p>	
+				<p>User Agents MUST ignore the aria-hidden state of any ancestor to the focused element from that point forward, resulting in the entire subtree to be exposed to assistive technologies. The ancestor aria-hidden markup will continue to be ignored, even once the element loses focus or is no longer focusable.  User agents MAY provide console messages to warn or flag aria-hidden errors to developers.</p>
```

Comment:
@scottaohara Here's a proposed re-wording for clarity (please feel free to tweak):

```suggestion
				<p>When an element receives focus, user agents MUST ignore the aria-hidden state of any ancestor of the focused element, thus re-exposing its entire descendent subtree to assistive technologies. The aria-hidden subtree MUST be permanently ignored, even once the element loses focus or is no longer focusable, if focus persists on the element (i.e., after all event listeners have been processed). User agents MAY provide console messages to warn or flag aria-hidden errors to developers.</p>
```